packages/base-map/src/styled.ts

```diff
@@ -100,14 +96,18 @@ export const LayerSelector = styled.aside`
       cursor: pointer;
     }
 
-    &:hover,
-    &.fake-mobile-hover {
+    /* :hover state is set on mouse over and on click on touch screens. */
```

Comment:
This is not happening for me

Reply:
Which device(s) are you using?

Reply:
*device(s) and browser(s)

Reply:
Had the same issue occur on Firefox for Android, Firefox for Mac, and Safari for Mac

Reply:
Reverted the hover stuff in https://github.com/opentripplanner/otp-ui/pull/578/commits/8c32bb4c8f950822c2ff724d31e0efaf3d25db14.